src/components/Menu/MenuItem/MenuToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
+import './MenuItem.scss';
+
+type MenuToggleProps = {
+    children: React.ReactNode;
+    direction: 'left' | 'right';
+    index: number;
+    onToggle?: (active: boolean) => void;
+    toggleStartLabel: string;
+    toggleStopLabel: string;
+}
+
+function MenuToggle(props: MenuToggleProps) {
+
+    const [status, setStatus] = useState(false);
```

Comment:
Please replace `status` with a better name which indicates the actual state and answers the boolean quetions, e.g. `isActive`, `isOpen` or whatever :)

Reply:
Done ✅ Renamed it to ```isActive```